### ISSUE TITLE
[CLI] unknown option '--yes'

### ISSUE THREAD

**Issue by jarrodwatts** [thirdweb-dev/js#3991]
I get this error when trying to use the CLI on a WSL environment.

```bash
npx thirdweb deploy

    $$\     $$\       $$\                 $$\                         $$\       
    $$ |    $$ |      \__|                $$ |                        $$ |      
  $$$$$$\   $$$$$$$\  $$\  $$$$$$\   $$$$$$$ |$$\  $$\  $$\  $$$$$$\  $$$$$$$\  
  \_$$  _|  $$  __$$\ $$ |$$  __$$\ $$  __$$ |$$ | $$ | $$ |$$  __$$\ $$  __$$\ 
    $$ |    $$ |  $$ |$$ |$$ |  \__|$$ /  $$ |$$ | $$ | $$ |$$$$$$$$ |$$ |  $$ |
    $$ |$$\ $$ |  $$ |$$ |$$ |      $$ |  $$ |$$ | $$ | $$ |$$   ____|$$ |  $$ |
    \$$$$  |$$ |  $$ |$$ |$$ |      \$$$$$$$ |\$$$$$\$$$$  |\$$$$$$$\ $$$$$$$  |
     \____/ \__|  \__|\__|\__|       \_______| \_____\____/  \_______|\_______/ 

 💎 thirdweb v0.14.12 💎

error: unknown option '--yes'
```

**Comment by jnsdls** [comment]
`--yes` this is a problem will take a look

**Comment by jnsdls** [comment]
@jarrodwatts this should be fixed in [5.45.1](https://github.com/thirdweb-dev/js/releases/tag/thirdweb%405.45.1) that just released, please re-open if you still see this issue
